main.js

```diff
@@ -0,0 +1,22 @@
+const popupPage = document.querySelector('.popup-page');
+const humbergur = document.querySelector('.pophumbergur');
+const popupLinks = document.querySelectorAll('.page-popup__link');
+const closeButton = document.querySelector('.popup-page__icon');
+function myToggler() {
+  if (popupPage.classList.contains('pop-menu')) {
+    popupPage.classList.remove('pop-menu');
+    humbergur.style.display = 'block';
+    popupPage.style.display = 'none';
+    closeButton.style.display = 'none';
+  } else {
+    popupPage.classList.add('pop-menu');
+    humbergur.style.display = 'none';
+    popupPage.style.display = 'block';
+    closeButton.style.display = 'flex';
+  }
+}
```

Comment:
- [ ] The page still scrolls while the menu is open. Please, prevent the page scroll while the menu is open as requested by the previous reviewer.

## Explanation

Scrolling the page with a popup open is bad!!! (almost always anyway)

The reason you would not want to allow scrolling though is that if your popup isn't fullscreen or is semi-transparent, users will see the content scroll behind the popup. In addition to that, when they close the popup they will now be in a different position on the page.

## Hints

A solution is that, is to add a CSS property to the body when the modal is open and remove it when it's closed.  This property would be:

```CSS
body {
    overflow: hidden;
}
```

Then, when closing the popup, you simply remove the class again, allowing scrolling again after the popup has closed.

Example with the function you already have:

```JAVASCRIPT
function myToggler() {
  if (popupPage.classList.contains('pop-menu')) {
    popupPage.classList.remove('pop-menu');
    humbergur.style.display = 'block';
    popupPage.style.display = 'none';
    closeButton.style.display = 'none';
    document.body.style.overflow = 'auto'; // NEW LINE
  } else {
    popupPage.classList.add('pop-menu');
    humbergur.style.display = 'none';
    popupPage.style.display = 'block';
    closeButton.style.display = 'flex';
    document.body.style.overflow = 'hidden'; //  NEW LINE
  }
}
```

### I tested it, worked for me (Firefox and Chrome).  Now go ahead and implement it 💪